scripts/release.js

```diff
@@ -55,13 +56,28 @@ const archive = async() => {
   );
 };
 
+const writeSegmentFile = () => {
+  console.log(process.env.SEGMENT_API_KEY)
```

Comment:
Would it be worth adding some prefix to this log like `with SEGMENT_API_KEY = `?

Reply:
temporary :sob: just trying to see if the logs on evergreen ever get here (they don't)